ParserPrinter.playground/Pages/Stock Ticker.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import Monoid
+import PartialIso
+import Syntax
+
+// Inspired by https://github.com/Flight-School/RegularExpressionDecoder
```

Comment:
recreated this regexp decoder thing with `Syntax`, which gave me the excuse to extract out a bunch of combinators into the library.